ApplicationLibCode/ProjectDataModel/Faults/RimFaultReactivationDataAccessorPorePressure.h

```diff
@@ -18,6 +18,7 @@
 
 #pragma once
 
+#include "RimFaultReactivationModel.h"
```

Comment:
Is including "RimFaultReactivationEnums.h" sufficient here?

Reply:
Fixed.